### PR TITLE
Added `selectedItem` and `selectedQuantity`

### PR DESCRIPTION
**Pull request by iaronaraujo**
Added `selectedItem` and `selectedQuantity` to the context so it would be possible to use `ProductQuantity` with the `ProductSummaryContext`.

### CODE REVIEW

File: react/ProductSummaryContext.tsx [hunk 1]
```diff
@@ -33,14 +35,23 @@ type SetLoadingAction = {
...
 
 export function reducer(state: State, action: Action) {
   switch (action.type) {
     case 'SET_PRODUCT': {
+      const product = action.args.product
       return {
         ...state,
-        product: action.args.product,
+        product: product,
+        selectedItem: product.sku
```
Comment (klzns) [inline react/ProductSummaryContext.tsx:55]: Get selectedItem based on the first sku with available quantity.

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+
+- `SET_QUANTITY` to the dispatch
```
Comment (klzns) [inline CHANGELOG.md]:
```suggestion
- `SET_QUANTITY` action
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+- `selectedItem` and `selectedQuantity` variables to the context
```
Comment (klzns) [inline CHANGELOG.md]:
```suggestion
- `selectedItem` and `selectedQuantity` to context state
```

**Review by klzns — CHANGES_REQUESTED** [review]

File: react/ProductSummaryContext.tsx (hunk 1 above)
Comment (jgfidelis) [reply react/ProductSummaryContext.tsx:55]: similar logic is used in ProductWrapper in `store` app, check there

**Review by klzns — APPROVED** [review]

**Pull request merged**